### ISSUE TITLE
Overwriting the appinstaller file with a new one results in malformed XML

### ISSUE THREAD

**Issue by rooju** [microsoft/MSIX-Toolkit#26]
I had written an appinstaller file by hand and then used this app to generate a new one.  In the app, I chose to save the file over the manually created one and that resulted in a malformed appinstaller file.

Here is the original manually authored XML

```
<?xml version="1.0" encoding="utf-8"?>
<AppInstaller
    xmlns="http://schemas.microsoft.com/appx/appinstaller/2018"
    Version="1.0.0.0"
    Uri="_uriremoved_">
	
    <MainPackage
        Name="FiddlerRepack"
        Publisher="CN=Rooju Chokshi, O=Rooju Corp, S=&quot;Washington C=US&quot;"
        Version="1.0.2.0"
        ProcessorArchitecture="x64"
        Uri="_uriremoved_"/>
		
	<UpdateSettings>
        <OnLaunch HoursBetweenUpdateChecks="12"/>
		<AutomaticBackgroundTask/>
		<ForceUpdateFromAnyVersion>true</ForceUpdateFromAnyVersion>
    </UpdateSettings>
</AppInstaller> 
```

And after I overwrote using the app this is what it wrote.  Note the trailing XML after the first closing AppInstaller tag.

```
<?xml version="1.0" encoding="utf-8"?>
<AppInstaller
  Version="1.0.2.0"
  Uri="D:\sample msix" xmlns="http://schemas.microsoft.com/appx/appinstaller/2017">
  <MainPackage
    Name="FiddlerRepack"
    Publisher="CN=Rooju Chokshi, O=Rooju Corp, S=&quot;Washington C=US&quot;"
    Version="1.0.2.0"
    ProcessorArchitecture="x64"
    Uri="D:\sample msix\FiddlerRepack_1.0.2.0_x64__4vhfqdnqb7ndr.msix" />
  <UpdateSettings>
    <OnLaunch
      HoursBetweenUpdateChecks="0" />
  </UpdateSettings>
</AppInstaller>unch HoursBetweenUpdateChecks="12"/>
		<AutomaticBackgroundTask/>
		<ForceUpdateFromAnyVersion>true</ForceUpdateFromAnyVersion>
    </UpdateSettings>
</AppInstaller>
```

**Comment by Huios** [comment]
Thanks for reporting this issue @rooju . We'll investigate and work on getting a fix out.

Tanaka

**Comment by Huios** [comment]
Fixed in version 1.2020.212.0 - https://github.com/microsoft/MSIX-Toolkit/releases/download/1.3.4/AppInstallerFileBuilder_1.2020.221.0_x86.msix
